src/buffer/out/textBuffer.cpp

```diff
@@ -184,6 +184,20 @@ TextBufferCellIterator TextBuffer::GetCellDataAt(const COORD at, const Viewport
     return TextBufferCellIterator(*this, at, limit);
 }
 
+// Routine Description:
+// - Retrieves read-only cell iterator at the given buffer location
+//   but restricted to operate only inside the given viewport.
+// Arguments:
+// - at - X,Y position in buffer for iterator start position
+// - limit - boundaries for the iterator to operate within
+// - until - X,Y position in buffer for last position for the iterator to read (inclusive)

```

Comment:
what is the difference between a limit and an until? Why can they be different?

Reply:
Updated the comment to be more clear. Also renamed...
- `limit` --> `bounds`
- `until` --> `limit`

Theoretically, you could have...
1. `bounds = bufferSize` & `limit = (3,3)`: iterate from `at` through `limit` (explores (4,3) --> (5,3) --> (6,3))
2. `bounds = (5,5)` & `limit = (3,3)`: iterate from `at` through `limit` _across `bounds`_ (explores (4,3) --> (5,3) --> (0,4))